Resprites the Tanto, again.

BLUNTFORCE420 opened the pull request:
Resprites the Tanto to better fit the Aurora artstyle, using a tweaked version of the new Everyman sprite. To make it visually distinct, it features a large, curved magazine and slightly longer, more complex-looking barrel.

![image](https://user-images.githubusercontent.com/65368125/177687250-46075df3-dcf9-4503-aac4-e7a63b962454.png)

BLUNTFORCE420 commented:
!review

TheDocOct commented:
![image](https://user-images.githubusercontent.com/22651198/177688372-9d5a3da6-63c5-4b19-bcb0-95cd0c01d27e.png)
For reference, the 'Tanto' below, the 'Everyman' above, another active resprite PR.

I personally think it looks too similar to the 'Everyman' resprite. I would have trouble distinguishing the two without taking a second to seriously think about which gun has which traits. The sprites are fine, but they aren't distinct enough from one another, and I think the current Tanto sprite is also quality and distinguishes it plenty.

BLUNTFORCE420 commented:
![image](https://user-images.githubusercontent.com/65368125/177897525-60b43d20-5ad0-4c9a-a1ce-2c8f8d2d3d75.png)
Update, modified action, added larger sights, made the barrel shroud brighter for more contrast, and trimmed the magwell.

KingOfThePing commented:
> ![image](https://user-images.githubusercontent.com/65368125/177897525-60b43d20-5ad0-4c9a-a1ce-2c8f8d2d3d75.png) Update, modified action, added larger sights, made the barrel shroud brighter for more contrast, and trimmed the magwell.

I like it. Looks cool!
